google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/GrpcResultSetTest.java

```diff
@@ -665,4 +665,161 @@ private void verifySerialization(
     Struct copy = reserialize(row);
     assertThat(row).isEqualTo(copy);
   }
+
+  @Test
+  public void getBoolean() {
+    consumer.onPartialResultSet(
+        PartialResultSet.newBuilder()
+            .setMetadata(makeMetadata(Type.struct(Type.StructField.of("f", Type.bool()))))
+            .addValues(Value.bool(true).toProto())
+            .addValues(Value.bool(false).toProto())
+            .build());
+    consumer.onCompleted();
+    resultSet.next();
+    assertThat(resultSet.getBoolean(0));
```

Comment:
is there assertTrue and assertFalse for use ?

Reply:
There is, but it's discouraged in new code according to google standards.

Reply:
However, I can probably clarify this a bit by using .isTrue()/.isFalse() at the end.

Reply:
@nithinsujir It's actually *necessary* to add `isTrue`/`isFalse` at the end.

You can think of this as sort of like a "test constructor". Saying `assertThat(x)` says you're testing `x` but doesn't perform any test yet. For example, a better way to write line 718 is `assertThat(l).isEqualTo(Long.MAX_VALUE)`

Reply:
Ah, thanks for the info , Michael. Yes, my latest commit fixes line 718 as well.